recipes/r-robustbase/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-robustbase
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "0.92_6"
+
+source:
+  fn: robustbase_0.92-6.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/robustbase_0.92-6.tar.gz
```

Comment:
so looks like other recipe for r packages in the existing bioconda recipe don't have md5 either. I wonder if it just doesn't include them automatically when you build your recipe with 'conda skeleton'?